daemon/test_support.go

```diff
@@ -87,13 +88,15 @@ func CreateTestBootnode(t *testing.T) (uint16, string) {
 		wg.Wait()
 	})
 
+	dataDir := t.TempDir()
+
 	conf := &bootnode.Config{
 		Env:           common.Development,
 		DataDir:       t.TempDir(),
 		Bootnodes:     nil,
 		P2PListenIP:   "127.0.0.1",
 		Libp2pPort:    0,
-		Libp2pKeyFile: common.DefaultLibp2pKeyFileName,
+		Libp2pKeyFile: path.Join(dataDir, common.DefaultLibp2pKeyFileName),
```

Comment:
This was a bug in my previous PR. Tests were littering network key files in the current working directory of the test.